app/poc-common/clinical-services/service-form/components/formWizard.js

```diff
@@ -239,19 +239,7 @@
     }
 
     function checkIn() {
-      var visitType = _.find($rootScope.defaultVisitTypes, function (o) {
```

Comment:
@edrisse there are different visitTypes for `clinic` and `pharmacy`, hope this is handled.

Reply:
Hum, will take a look at that. Don't merge yet.

Reply:
@Fixed on the back-end. You can proceed with the review.